data/Strings.lua

```diff
@@ -406,60 +406,60 @@ local Strings = {
         }
     },
     ["pl-pl"] = { -- Polish
-        thirst = "Thirst",
-        hunger = "Hunger",
-        fatigue = "Fatigue",
+        thirst = "Pragnienie",
+        hunger = "Głód",
+        fatigue = "Zmęczenie",
         step = {
             thirst = {
-                "You are hydrated, ",
-                "Low Dehydration, ",
-                "Dehydration, ",
-                "Severe Dehydration, ",
+                "Odpowiednie nawodnienie, ",
```

Comment:
`Jesteś nawodniony` seems to be straightforward translation - same in other cases like:
- `Jesteś najedzony`
- `Jesteś wypoczęty`

Reply:
"Jesteś nawodniony", "Jesteś najedzony", "Jesteś wypoczęty" is more precise translation (1:1), but in Polish it only refers to male gender. I was looking for something more neutral.

Reply:
Ok, that's reasonable.